packages/edit-site/src/components/dataviews/view-actions.js

```diff
@@ -250,29 +237,32 @@ function SortMenu( { fields, view, onChangeView } ) {
 										} }
 									>
 										{ info.label }
-									</DropdownMenuItemV2>
+									</DropdownMenuRadioItemV2Ariakit>
 								);
 							}
 						) }
-					</DropdownSubMenuV2>
+					</DropdownMenuV2Ariakit>
 				);
 			} ) }
-		</DropdownSubMenuV2>
+		</DropdownMenuV2Ariakit>
 	);
 }
 
 export default function ViewActions( { fields, view, onChangeView } ) {
 	return (
-		<DropdownMenuV2
-			label={ __( 'Actions' ) }
```

Comment:
This `label` prop wasn't actually doing anything. The correct place for it is on the `Button` component used as a trigger